pkg/config/install_requirements.go

```diff
@@ -243,6 +243,22 @@ type IngressConfig struct {
 	DomainIssuerURL string `json:"domainIssuerURL,omitempty"`
 }
 
+// BuildPackConfig contains build pack info
+type BuildPackConfig struct {
+	// Location contains location config
+	Location LocationConfig `json:"location,omitempty"`
+}
+
+// LocationConfig contains buildpack location
+type LocationConfig struct {
```

Comment:
After the discussion with @jstrachan yesterday, I think that BuildPackLibrary is the best term for what we are trying to model here. 

It would btw, be nice to add a test to _install_requirements_test.go which shows how these structs serialize. The naming of these structs is one thing, but how this actually is written into jx-requirements.yaml is driven by the annotations.

Reply:
Did we not say we are going to use the term library?

Reply:
yup PR has been updated, just pushed passing test up